sensorlib/src/util/CarlaUtils.py

```diff
@@ -45,17 +45,16 @@ def get_actor_angular_velocity(carla_actor):
         return np.deg2rad(angular_velocity_degpersecond)
 
     @staticmethod
-    def get_actor_rotation_matrix(carla_actor):
+    def get_actor_orientation(carla_actor):
```

Comment:
It might be more descriptive to rename this to something like `get_actor_roll_pitch_yaw` since there are several orientation data structures.